packages/newspringchurchapp/src/prayer/data/mutations/savePrayer.js

```diff
@@ -4,6 +4,21 @@ export default gql`
   mutation SavePrayer($nodeId: String!) {
     savePrayer(nodeId: $nodeId) {
       id
+      firstName
```

Comment:
Is all this data necessary? In my mind I would think all you would need to save a prayer would be an Id?

Reply:
It's needed for the cache update function

Reply:
I guess I don't understand the cache update function enough. Just more for me to learn. I'm good with this though.